js/components/plates.js

```diff
@@ -65,7 +65,9 @@ export default class Plates extends PureComponent {
       renderEulerPoles: config.renderEulerPoles,
       renderBoundaries: config.renderBoundaries,
       renderLatLongLines: config.renderLatLongLines,
-      snapshotAvailable: false
+      snapshotAvailable: false,
+      plateDensities: {},
+      plateColors: {}
```

Comment:
Ended up needing state for the plate colors to pass to the density selector, does this seem like the best way to do that?

Reply:
It seems fine to me. Otherwise, we could keep `plates` array which would be sorted by densities and would include more information (e.g. ids, colors, and whatever is necessary in the future). Either way is fine with me.